test/testcafe/spec-en-leaks/EnglishLeaks_spec.js

```diff
@@ -19,10 +19,15 @@ const ignoredMocks = [
   'error-internal-server-error.json',
   'error-forgot-password.json',
   'authenticator-verification-select-authenticator.json',
+  'authenticator-verification-okta-verify-signed-nonce-loopback.json',
```

Comment:
Why are we adding this back to the ignore list ?

Reply:
Fxed